components/02-elements/table/TableCell.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <component
+    :is="tag"
+    role="gridcell"
+  >
+    <slot />
+  </component>
+</template>
+
+<script>
+  export default {
+    props: {
+      tag: {
+        type: String,
+        default: 'td'
+      }
+    }
+  }
+</script>
+
+<style lang="scss">
+  $table__padding                  : $spacer--medium !default;
+  $table__background               : $white !default;
+  $table__border                   : 1px solid $gray-light !default;
+  $table__border-radius            : $border-radius !default;
+  $table__th-background            : $gray-lightest !default;
+  $table__td-background            : $white !default;
+  $table__tfoot-background         : $gray-lightest !default;
+  $table__background-odd--odd-even : $white !default;
+  $table__background-even--odd-even: #f9f9f9 !default;
```

Comment:
can we use any color from global variables? It would be great to use one of them or create a new one on the global scope.